src/screens/AddSiteFormScreen.js

```diff
@@ -465,12 +554,29 @@ const styles = {
         display: 'flex',
         flexDirection: 'row',
         justifyContent: 'space-around'
+    },
+    pickerStyle: {
+        width: 250,
+        alignSelf: 'center'
+    },
+    cellProviderPickerStyle:{
+        width: 90
+    },
+    cellStrengthPickerStyle:{
+        width: 130
+    },
+    cellServiceContainerStyle: {
+        width: 250,
+        alignSelf: 'center',
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        alignContent: 'center'
```

Comment:
Put both pickers on the same line so they read like english as you select them, for example `Verizon` `LTE`